code/datums/gamemodes/crash.dm

```diff
@@ -200,6 +200,12 @@
 	var/larva_surplus = (get_total_joblarvaworth() - (num_xenos * xeno_job.job_points_needed )) / xeno_job.job_points_needed
 	if(larva_surplus < 1)
 		return //Things are balanced, no burrowed needed
+	for(var/mob/living/carbon/human/H AS in GLOB.human_mob_list)
+		if(!H.job)
+			continue
+		var/obj/item/card/id/user_id =  H.get_idcard()
+		for(var/i in user_id.marine_points)
+			user_id.marine_points[i] += 1
```

Comment:
I'm not entirely sure if you're doing what you intent to do? This only gives points if there are no larva etc, since this is after all the other checks above.

Reply:
I think it does what I intend to do. I thought no larvas are covered by
`var/num_xenos = xeno_hive.get_total_xeno_number() + stored_larva
	if(!num_xenos)
		xeno_job.add_job_positions(1)
		return`

Reply:
Yeah I mean, is it intended to only work under those conditions instead of always? And if so, why?
Wasn't clear is all.

Reply:
It's intended to only work if there is more than 0 xenos and autobal larva gets generated and not always. Otherwise marines would get spammed with free resources and that would be a little too strong.